Formula/ibex.rb

```diff
@@ -1,8 +1,20 @@
 class Ibex < Formula
   desc "C++ library for constraint processing over real numbers."
   homepage "http://www.ibex-lib.org/"
-  url "http://www.ibex-lib.org/sites/default/files/ibex-2.1.16.tar.gz"
-  sha256 "d92ff32f14d27ad7b390ae693beb311b58cf6babccac85bbdaa5f5d0b8648845"
+  head "https://github.com/ibex-team/ibex-lib.git", :branch => "master"
+
+  stable do
+    url "https://github.com/ibex-team/ibex-lib/archive/ibex-2.2.0.tar.gz"
+    sha256 "fa6f281d5f28db11fa8715569937b4d19bd24ab744d0c7f21674cc1552d674ee"
+    # Patch the optional affine plugin build script
```

Comment:
Please leave a gap between the end of the initial URL/SHA block and the first patch, Thanks!